Common/Util/ResolutionTimeSpan.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace QuantConnect.Util
+{
+    /// <summary>
+    /// This class allows implicit conversions between Resolution and TimeSpan
+    /// </summary>
+    public class ResolutionTimeSpan
```

Comment:
Please add unit tests for this class. Also included under test should be use cases involving python.

Reply:
I think this type is probably better defined as a `struct` since it represents a value (timespan is `struct` and enums are `structs` as well).

Reply:
No idea how to write tests including use cases involving python.